core/src/main/java/org/vertexium/FetchHintsBuilder.java

```diff
@@ -44,6 +48,9 @@ public FetchHints build() {
         if (!isIncludeProperties() && isIncludePropertyMetadata()) {
             includeAllProperties = true;
         }
+        if (!includeEdgeIds && !includeEdgeVertexIds) {
```

Comment:
Technically not an error if they also excluded edge refs.

Reply:
good point. I'll make that change